src/pages/ProfilesExplorerView/components/SceneExploreServiceLabels/components/SceneGroupByLabels/components/SceneLabelValuesGrid/infrastructure/buildLabelValuesGridQueryRunner.ts

```diff
@@ -0,0 +1,20 @@
+import { SceneQueryRunner } from '@grafana/scenes';
+
+import { PYROSCOPE_DATA_SOURCE } from '../../../../../../../infrastructure/pyroscope-data-sources';
```

Comment:
**suggestion (non-blocking):** this is importing stuff from other component's infrastructure. Should it be part of "shared"

Reply:
I'll take note for a future PR